bin/lerna.js

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env node
 "use strict";
+const resolveCwd = require("resolve-cwd");
```

Comment:
I like the (more recent) [import-local](https://github.com/sindresorhus/import-local#usage) pattern, which encapsulates the tricky stuff for us:

```js
const importLocal = require("import-local");

if (importLocal(__filename)) {
    require("npmlog").verbose("cli", "using local version of lerna");
} else {
    require("../lib/cli")().parse(process.argv.slice(2));
}
```